src/v0/destinations/adobe_analytics/transform.js

```diff
@@ -19,44 +19,26 @@ const {
   ConfigurationError,
 } = require('../../util/errorTypes');
 
```

Comment:
If there is no comment which points to the adobe_analytics documentation, please include it as part of this PR, otherwise feel free to ignore & resolve this comment

Reply:
Added some inline documentation references.